doc/man3/OSSL_DECODER_CTX_new_for_pkey.pod

```diff
@@ -120,10 +123,6 @@ failure.
 Parts of the function names are made to match already existing OpenSSL
 names.
 
-B<EVP_PKEY> in OSSL_DECODER_CTX_new_by_EVP_PKEY() matches the type name,
-thus making for the naming pattern B<OSSL_DECODER_CTX_new_by_I<TYPE>>() when
-new types are handled.
-
```

Comment:
You can remove the whole note, as it's no longer true per se

Reply:
Done